contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -693,8 +693,6 @@ update_DropRoleStmt(Node *n, const char *role)
 		if (strcmp(tmp->rolename, "is_role") == 0)
 			stmt->roles = list_delete_cell(stmt->roles, list_head(stmt->roles));
 
-		pfree(tmp);
```

Comment:
we should add test case for this. We should have a way to test functions like sys.remove_babelfish() with different input parameters. May be to have such tests run strictly in the end after regular tests are done?

Reply:
Tracking effort of adding automated test as separate task.